backend/internal/controller/server/msg_handler.go

```diff
@@ -66,3 +69,36 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 func (s server) ResetScoreHandler(w http.ResponseWriter, r *http.Request) {
 	s.game.ResetScore()
 }
+
+// ManipulateScoreHandler is a handler for manipulation of the score.
+// Incoming URL should be in the format: '/goal?action=[add/sub]&team=[1/2]'.
+// Team ID 1 stands for white and 2 for blue.
+func (s server) ManipulateScoreHandler(w http.ResponseWriter, r *http.Request) {
+
+	team := r.URL.Query().Get(attributeTeam)
+
+	teamID, err := strconv.Atoi(team)
+	if err != nil || !isValidTeamID(teamID) {
+		handleError(w, http.StatusBadRequest, "Team ID has to be a number either 1 or 2")
+		return
+	}
+
+	switch action := r.URL.Query().Get(attributeAction); action {
+	case "add":
+		s.game.AddGoal(teamID)
+	case "sub":
+		// TODO: s.game.SubGoal(teamID)
```

Comment:
Maybe this should send some error message to the user?
`handleError`

Reply:
We are logging the error message and showing it on the connection, where else should we displaying it?

Reply:
When I as a user, send a request with `sub`, what do I receive? 200 with empty response? or some error message?

Reply:
The sub method will be added as soon as this PR will be accepted.

Reply:
It won't be accepted if it introduces an invalid state ;)
`200` response means that everything went ok. This is a server error - unimplemented.